test/test_sdam_monitoring_spec.py

```diff
@@ -186,7 +186,7 @@ class NoopMonitor(Monitor):
             def _run(self):
                 time.sleep(0.05)
 
-        m = single_client(h=scenario_def['uri'], p=27017,
+        m = MongoClient(host=scenario_def['uri'], port=27017,
                           event_listeners=[self.all_listener],
                           _monitor_class=NoopMonitor)
```

Comment:
Please fix the indentation of these lines.

Reply:
Done.